packages/office-ui-fabric-react/src/components/DetailsList/DetailsList.tsx

```diff
@@ -1,5 +1,6 @@
 import * as React from 'react';
 import {
+  BaseComponent,
   EventGroup,
```

Comment:
Because you're removing usages of EventGroup, you need to remove the import to make tslint happy.